recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class PahoMqttcConan(ConanFile):
+    name = "paho-mqtt-c"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/eclipse/paho.mqtt.c"
+    topics = ("MQTT", "IoT", "eclipse", "SSL", "paho", "C")
+    license = "EPL-1.0"
+    description = """The Eclipse Paho project provides open-source client implementations of MQTT
+    and MQTT-SN messaging protocols aimed at new, existing, and emerging applications for the Internet
+    of Things (IoT)"""
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake", "cmake_find_package"
```

Comment:
The `cmake_find_package` generator is not needed. `paho-mqtt-c` does not use `find_package`.